testing/benchmark/generate_metrics.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+#
+# This script is expected to run from $ENGINE_PATH/src/out/host_release/
+# it is currently used only by automation to collect and upload metrics.
+
```

Comment:
In both scripts, please add `set -ex` to catch failures.

Reply:
Updated.